src/progress-steps/styled-components.js

```diff
@@ -130,7 +130,7 @@ export const StyledContentTail = styled(
       ':after': {
         content: '""',
         display: 'inline-block',
-        height: '100%',
+        height: 'calc(100% + 14px)',
```

Comment:
please use a theme `sizing` value rather than the `14px` so that it can scale based on theme values. look up the `creator.js` file in the project to view available values